aries-site/src/pages/templates/page-layouts.mdx

```diff
@@ -1,150 +1,159 @@
-import { useContext } from 'react';
-import { Box, Grid, ResponsiveContext, Text } from 'grommet';
-import { ContentPreviewCard } from '../../components';
-import {
-  HeaderOnlyExample,
```

Comment:
Since we are shifting away from these examples -- should we delete the files?

Reply:
I debated it. I ended up keeping them until we get the rest of content fleshed out and start hearing feedback from stakeholders.

I can see sticky header vs. non-sticky as being a pattern folks will ask about.

Reply:
`I can see sticky header vs. non-sticky as being a pattern folks will ask about, so thought it was premature to delete.`

I agree this is a good idea to keep.